test/core/componentTests.ts

```diff
@@ -403,6 +403,33 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
+  it("rendering to a new svg detaches the component", () => {
```

Comment:
I actually gave it a thought. I put it in the same category with the other detach() methods on the component.